certification/README.md

```diff
@@ -0,0 +1,2 @@
+# certification
+Software Carpentry certifications
```

Comment:
Everything should be documented (though we're rather lax about doing 
that ourselves sometimes).  @pbanaszkiewicz can you review the PR when 
it comes in?